cumulus_library/cli.py

```diff
@@ -338,5 +334,9 @@ def main(cli_args=None):
     return run_cli(args)
 
 
+def main_cli():  # called by the generated wrapper scripts
+    main()
```

Comment:
This is so that the generated wrappers can avoid a return (which gets printed) and the unit tests can keep the return of the builder.